samples/dotnet/kernel-syntax-examples/Example16_OpenApiSkill.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.CoreSkills;
+using Microsoft.SemanticKernel.Orchestration;
+using Microsoft.SemanticKernel.Orchestration.Extensions;
+using Microsoft.SemanticKernel.Skills.OpenAPI.Extensions;
+using Microsoft.SemanticKernel.Skills.OpenAPI.Skills;
+using RepoUtils;
+
+namespace KernelSyntaxExamples;
+internal class Example16_OpenApiSkill
+{
+    public static async Task RunAsync()
+    {
+        await GetSecretFromAzureKeyValutAsync();
+    }
+
+    public static async Task GetSecretFromAzureKeyValutAsync()
+    {
+        Console.WriteLine("======== Planning - Create and Execute Azure Plan ========");
+        var kernel = InitializeKernelAndPlanner(out var planner);
+
+        //Use OpenApi skill from folder
+        //string folder = RepoFiles.SampleSkillsPath();
+        //kernel.ImportOpenApiSkillFromDirectory(folder, "AzureKeyVaultSkill");
+
+        //Use OpenApi skill from Skills.OpenAPI package
+        kernel.RegisterOpenApiSkill(LocalOpenApiSkillsProvider.LoadFroResource(LocalOpenApiSkillsProvider.AzureKeyVault), LocalOpenApiSkillsProvider.AzureKeyVault);
+
+        var plan = await kernel.RunAsync("Load 'test-secret' from Azure KeyValut available at https://dev-tests.vault.azure.net using GetSecret function.", planner["CreatePlan"]);
```

Comment:
In the future I would recommend not using the planner for this particular example, so as to keep the concept count limited to what we are trying to show.